src/components/ui/TaskCard.jsx

```diff
@@ -0,0 +1,26 @@
+import classNames from 'classnames';
+import TaskFooter from 'components/ui/TaskFooter';
+import PropTypes from 'prop-types';
+
+function TaskCard({ details = '', createdAt = '', isCompleted = false }) {
+  const taskCardClasses = classNames({
+    task__details: true,
```

Comment:
if it's always true, we don't need to write it like this, instead we can write like 
```suggestion
    const taskCardClasses = classNames(task__details, {
    .......
```

Reply:
Changed as suggested. thanks